### DISCUSSION TITLE
pi-Base Community Meeting: April 21 4pm Eastern

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#296] [Announcements]
- Schedule onboarding for new members
- Discussion: use GitHub discussions rather than email for written communication?
    - (Slack a possibility as well)
    - Port certain Issues to discussions
- Survey paper proposal: between T1 and T2
- [Murray Bell's Booelan Spaces Machine](https://github.com/pi-base/data/issues/269)
- Naming conventions for properties

**Comment by prabau** [comment]
Forgot to mention in the meeting, but if improvements to the web interface need to be prioritized, I think it would be beneficial and easier to add number of entries returned in a search (as we had discussed previously) before adding graphical interface improvements, which would be a lot more involved.

**Reply by StevenClontz** [reply]
I may have already done this. Or it's quite easy to do anyway. Once I finally can get the updated viewer app deployed, please remind me if this was forgotten in that upgrade.
